src/main/scala/github/gphat/censorinus/dogstatsd/Encoder.scala

```diff
@@ -62,14 +62,15 @@ object Encoder extends MetricEncoder {
 
   def encodeEvent(sc: EventMetric): String = {
     val sb = new StringBuilder()
+    val escapedText = sc.text.replace("\n", "\\\\n")
```

Comment:
Since we had them fix it, I don't think this is necessary. Do you think we should?